installation/README.md

```diff
@@ -1,11 +1,9 @@
 # Installation
 
-Some simple guides to setting up the software on your Raspberry Pi.
+We recommend using [Raspberry Pi Imager](https://www.raspberrypi.org/software/) to install an operating system on your Raspberry Pi. As well as Raspberry Pi OS, you can use it to install other operating systems such as Ubuntu or LibreELEC (Kodi media centre).
 
-Beginners should start with [NOOBS](noobs.md), which gives the user a choice of operating system from the standard distributions. The recommended distribution for normal use is Raspberry Pi OS. Alternatives are available, such as LibreELEC (Kodi media centre) or Arch Linux.
+* You can find detailed help with installing OS images [here](installing-images/). For further assistance, check out our [forums](https://www.raspberrypi.org/forums/).
 
-## Contents
+* [Further information about SD cards](sd-cards.md).
```

Comment:
Feels like this should be something like "You can find more detailed information about SD card requirements **here**" and be merged with the previous "You can find..." sentence, rather than having separate bullet points? :shrug: 
(just looks a bit out-of-context otherwise, IMHO)

Reply:
Again, I don't think it's worth spending too much time working on this page. Your wording is perhaps better, but it's also more wordy so I don't think it's worth changing.

Reply:
Fair enough, I just thought that https://github.com/raspberrypi/documentation/blob/4839e89445ee97293a399a3c6abf46dbd806f008/installation/README.md looked a bit odd - but maybe that's just me.